srcalc/src/main/java/gov/va/med/srcalc/service/EditVariable.java

```diff
@@ -7,7 +7,7 @@
 
 /**
  * A command object that updates a {@link AbstractVariable}'s properties. See
- * {@link AdminService#updateVariable(AbstractVariable)} for a primary usage.
+ * {@link AdminService#updateVariable(EditVariable)} for a primary usage.
```

Comment:
I think explaining why an EditVariable class was used to update an AbstractVariable would help here.